app/src/main/res/layout/activity_create_key_gear_item.xml

```diff
@@ -91,7 +91,7 @@
         android:layout_marginEnd="@dimen/base_margin_quadruple"
         android:layout_marginBottom="@dimen/base_margin_quadruple"
         android:background="@drawable/button_primary"
-        android:backgroundTint="?colorSecondary"
+        android:backgroundTint="@color/colorLargeButton"
```

Comment:
Use a color reference. Also, `colorLargeButton` should be removed in favor of `colorButton`